src/crow/main.cpp

```diff
@@ -0,0 +1,51 @@
+/* example usage of crow-cpp http://crowcpp.org/ */
+#include "crow.h"
+
+class a : public crow::returnable
+{
+public:
+  a() : returnable("text/plain"){};
+
+  std::string dump() const override
+  {
+    return "ABCDF";
+  }
+};
+
+int main()
+{
+  constexpr uint16_t port = 8080;
+  std::vector<int> values;
+  crow::SimpleApp app;
+
+  CROW_ROUTE(app, "/")
+  ([]() {
```

Comment:
in the other lambdas, you omitted the empty parentheses ;-)

Reply:
fixed :)

Reply:
parentheses here...